packages/usa-date-picker/src/index.js

```diff
@@ -80,30 +80,41 @@ const CALENDAR_YEAR_FOCUSED = `.${CALENDAR_YEAR_FOCUSED_CLASS}`;
 
 const VALIDATION_MESSAGE = "Please enter a valid date";
 
-const MONTH_LABELS = [
-  "January",
-  "February",
-  "March",
-  "April",
-  "May",
-  "June",
-  "July",
-  "August",
-  "September",
-  "October",
-  "November",
-  "December",
-];
-
-const DAY_OF_WEEK_LABELS = [
-  "Sunday",
-  "Monday",
-  "Tuesday",
-  "Wednesday",
-  "Thursday",
-  "Friday",
-  "Saturday",
-];
+const MONTH_DATE_SEED = Array.from({ length: 12 }).map(
+  (_, i) => new Date(0, i)
+);
+
+const DAY_OF_WEEK_DATE_SEED = Array.from({ length: 7 }).map(
+  (_, i) => new Date(0, 0, i)
+);
+
+let monthLabels;
+let dayOfWeeklabels;
+let dayOfWeeksAbv;
+
+function setCalendarLabels() {
+  monthLabels = MONTH_DATE_SEED.map((date) =>
+    date.toLocaleString("es", { month: "long" })
```

Comment:
**issue**: This is Spanish by default and there's no way for devs to set dynamically.